onnxruntime/core/optimizer/conv_activation_fusion.cc

```diff
@@ -121,6 +121,9 @@ Status ConvActivationFusion::ApplyImpl(Graph& graph, bool& modified, int graph_l
         graph_utils::FinalizeNodeFusion(graph, {conv_node, act_node}, fused_conv);
         modified = true;
       } else if (graph_utils::IsSupportedOptypeVersionAndDomain(next_node, "Add", {6, 7, 13, 14})) {
+        if (next_node.GetOutputEdgesCount() == 0) {
```

Comment:
Can you check if there are other places in this file that need a more robust check?

Reply:
Added check for dependents between Conv and Add, plus another UT model to make the graph being left intact is there are more than one downstream nodes of Add.